tests/robot-tests/tests/admin/bau/adopt_methodology.robot

```diff
@@ -40,9 +47,15 @@ Create adopting publication
     user adds headlines text block
     user adds content to headlines text block    Headline text block text
 
-Adopt a Methodology
+Check that an unpublished methodology cannot be adopted
     user navigates to methodologies on publication page    ${ADOPTING_PUBLICATION_NAME}
+    user waits until page contains link    Adopt an existing methodology
+    user clicks link    Adopt an existing methodology
+    user waits until h2 is visible    Adopt a methodology
+    user checks page does not contain    ${OWNING_PUBLICATION_NAME_2}
 
+Adopt a published Methodology
+    user navigates to methodologies on publication page    ${ADOPTING_PUBLICATION_NAME}
     user waits until page contains link    Adopt an existing methodology
     user clicks link    Adopt an existing methodology
     user waits until h2 is visible    Adopt a methodology
```

Comment:
These lines can be removed now, since we already navigated to the adopt a methodology page in the previous test case.